RadarSDK/Radar.m

```diff
@@ -1295,9 +1295,9 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
-+ (BOOL)isTestKey {
++ (BOOL)canFlushLogs {
     NSString *publishableKey = [RadarSettings publishableKey];
-    if ([publishableKey hasPrefix:@"prj_test_pk"] || [publishableKey hasPrefix:@"org_test_pk"] || [RadarSettings userDebug]) {
+    if ([publishableKey hasPrefix:@"prj_test_pk"] || [publishableKey hasPrefix:@"org_test_pk"] || [RadarSettings sdkConfiguration].logLevel == RadarLogLevelDebug) {
```

Comment:
```suggestion
    if ([publishableKey hasPrefix:@"prj_test_pk"] || [publishableKey hasPrefix:@"org_test_pk"] || [RadarSettings userDebug] || ([RadarSettings sdkConfiguration].logLevel && [RadarSettings sdkConfiguration].logLevel != RadarLogLevelNone)) {
```

Reply:
^if we're remotely setting a log level, we should be flushing, so long as it's not log level none. Also need to flush if the user has been manually overwritten